shared/actions/deeplinks.tsx

```diff
@@ -135,6 +136,16 @@ const handleKeybaseLink = (action: DeeplinksGen.HandleKeybaseLinkPayload) => {
   ]
 }
 
+const handleServiceAppLink = (action: EngineGen.Keybase1NotifyServiceHandleKeybaseLinkPayload) => {
+  const link = action.payload.params.link
+  if (action.payload.params.deferred && !link.startsWith('keybase://team-invite-link/')) {
```

Comment:
the referrer link coming from the play store is potentially malicious. Whoever reviews the tsx part should make sure that strings which start with `keybase://team-invite-link/` cannot have unintended consequences during their handling.

Reply:
To address this: `actions/deeplinks.tsx` will be changed if I get to finish this, as of right now there is no code for handling deeplinks if you're logged out.